### DISCUSSION TITLE
Plugin monitor not showing anymore + buffer settings not retained

### DISCUSSION THREAD

**Discussion by MrPisquan** [apohl79/audiogridder#1127] [General]
Hi,
AudioGridder has been in my workflow constantly for several months, and it is an unmissable part of my everyday work.
It is a revolutionary plugin, and I couldn't be more grateful for it.
For some reason, though, recently, the plugin monitor isn't showing up, and also, there is no way to set permanently the latency at a different buffer size than 2 blocks. I could set the latency manually, of course, but it is not retaining the setting on the next session.
Nothing that could affect my work, anyway.
Is there any suggestion about it?
Thank you very much for your precious work and I'm looking forward to furthering improvements and seeing the most awaited next official release.

**Comment by MrPisquan** [comment]
Hi,
Do you have any suggestions about this topic?
I would really appreciate that.
Thank you again

**Comment by apohl79** [comment]
Can you open the monitor manually using the tray app?

Regarding buffer settings, have you enabled individual plugin buffer settings? The buffer setting will also be stored in the DAW session/project. Does that work for you?

**Comment by MrPisquan** [comment]
Hi Andreas,Thank you very much for reaching out.The tray app doesn’t open. In task manager shows for less than a second. Uninstalling and installing again doesn’t help…Oh yes, I forgot to have enabled the individual plugin buffer settings, sorry! Now obviously, AG retains the buffers settings; thank you for reminding me.I don't recall any event associated with the disappearance of the plugin monitor, though, even if my memory is not something to be blindly relying on seeing the previous topic! :-)Thank you very much, your work has changed my life!!!All the bestFedericoOn 12 Jan 2023, at 07:26, Andreas Pohl ***@***.***> wrote:﻿
Can you open the monitor manually using the tray app?
Regarding buffer settings, have you enabled individual plugin buffer settings? The buffer setting will also be stored in the DAW session/project. Does that work for you?

**Reply by apohl79** [reply]
ok, so something is wrong with the tray app. it gets started by the ag plugin. please check that you have not disabled it in the plugin settings under "user interface". if it is enabled for you, it seems that the plugin can't start it for some reason or it crashes. can you check the plugin log file for an error running the tray app? just load one ag plugin, so the log does not get swamped.

if the plugin can run the tray app, please check the tray app logfile for errors.

https://audiogridder.com/bug-report-diagnostics-locations/

thanks!

**Reply by MrPisquan** [reply]
Hi,

Thank you very much for the support.

The plugin log file says it failed to start the tray app. Please take a look at the files I've attached.

Thank you very much again

[AudioGridderPlugin_2023-01-12_20-16-32.zip](https://github.com/apohl79/audiogridder/files/10405695/AudioGridderPlugin_2023-01-12_20-16-32.zip)

**Reply by MrPisquan** [reply]
p.s. I haven't disabled the tray app from the user interface option of the plugin.

**Reply by apohl79** [reply]
The plugin is not able to launch the tray app under `C:/Program Files/AudioGridderPluginTray/AudioGridderPluginTray.exe`. Does the file exists at that location? Can you launch it manually (it will close after a few seconds)? If the file does not exists under that location, you have to reinstall the plugin.

**Comment by MrPisquan** [comment]
Hi,The file exists. When I launch it manually (even when it is run as admin), it doesn't show. I can see it for less than a second under task manager, though.I have reinstalled it several times already before writing the post and even after.Thank you!On 13 Jan 2023, at 16:28, Andreas Pohl ***@***.***> wrote:﻿
The plugin is not able to launch the tray app under C:/Program Files/AudioGridderPluginTray/AudioGridderPluginTray.exe. Does the file exists at that location? Can you launch it manually (it will close after a few seconds)? If the file does not exists under that location, you have to reinstall the plugin.

**Reply by apohl79** [reply]
That means the executable works. Not sure, what the issue is though. Sounds like some permissioning issue to me.

**Comment by MrPisquan** [comment]
Hi Andreas,
Could it be related to some Windows update? Does the Plugin Monitor need some Windows service? 
There is also a weird behaviour of the server. When I quit it, it keeps restarting all over, opening  infinite instances of it
Thank you for the support!

**Reply by apohl79** [reply]
Not sure tbh. Have never seen this before. My test setup is still on win10 also.

**Comment by MrPisquan** [comment]
Windows 11 here...
But your suspicion about some permission issue is absolutely right: I have started Cubase as administrator and the plugin monitor has shown up!
I also deleted the AG folders, reinstalling both the server and the plugin and now the problem about the "never quitting" server is fixed... 
Still, I don't understand, but I am relieved. AudioGridder is a sensitive matter since I have adopted it massively in every production. Every single weirdness has a huge importance to me!
So thank you again Andreas, I really am looking forward to seeing the next non-beta release, and along with me, there are a lot of people longing for it!
AG is pure genius.
All the best

**Reply by apohl79** [reply]
Ok thats good. So it is solvable and it should be possible to run cubase as a non-admin user after all. 

> So thank you again Andreas, I really am looking forward to seeing the next non-beta release, and along with me, there are a lot of people longing for it!
> AG is pure genius.

thanks! :-)

**Reply by apohl79** [reply]
Btw: you could also run the tray app manually and pass the parameter `-keeprunning` to it. In that case a plugin can just connect to it, even if it can't run the executable.

**Comment by MrPisquan** [comment]
Very interesting, if I’m in trouble again with it I’ll follow your suggestion. 

Thank you Andreas
